.gitignore

```diff
@@ -25,4 +25,5 @@ build/
 # dist/
 
 ## Key Files
-*.pem
\ No newline at end of file
+*.pem
```

Comment:
to be extra safe, make sure your pem keys aren't located within your folders. Best to keep it elsewhere